transform/operations.go

```diff
@@ -246,6 +246,23 @@ func (t *timeParse) transform(raw interface{}) (interface{}, error) {
 	return parsedTime.Format(t.Args["layout"]), nil
 }
 
+type currentTime struct {
+	Args map[string]string
```

Comment:
lower case.

Reply:
why is this one lower case when all of the other operations are upper case?

Reply:
commented below. but these should all be lower case. probably an oversight.